packages/jest/src/utils/versions.ts

```diff
@@ -4,3 +4,5 @@ export const jestTypesVersion = '27.0.2';
 export const tslibVersion = '^2.0.0';
 export const tsJestVersion = '27.0.5';
 export const babelJestVersion = '27.2.3';
+export const swcCoreVersion = '1.2.85';
```

Comment:
nickpick: bump to 1.2.119 
this was probably the latest when you first made the PR sorry!